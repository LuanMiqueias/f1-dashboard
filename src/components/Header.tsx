import Link from "next/link";
import styles from "../styles/components/Header.module.css";

const Header = () => {
  return (
    <header className={styles.container}>
      <div className={styles.content}>
        <Link href="/">
          <img src="logo-f1.svg" alt="logo" className={styles.logo} />
        </Link>
        <p className={styles.year}>{new Date().getFullYear()}</p>
      </div>
    </header>
  );
};

export default Header;
