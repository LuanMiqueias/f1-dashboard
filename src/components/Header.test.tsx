import { render } from "@testing-library/react";
import Header from "./Header";

it("Should math snapshot", () => {
  const { container } = render(<Header />);
  expect(container.firstChild).toMatchSnapshot();
});
