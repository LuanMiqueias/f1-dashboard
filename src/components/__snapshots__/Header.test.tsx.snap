// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should math snapshot 1`] = `
<header
  class="container"
>
  <div
    class="content"
  >
    <img
      alt="logo"
      class="logo"
      src="logo-f1.svg"
    />
    <p
      class="year"
    >
      2021
    </p>
  </div>
</header>
`;
